code/modules/vore/smoleworld/smoleworld_vr.dm

```diff
@@ -0,0 +1,407 @@
+	visible_message("<span class='danger'>\The [src] explodes into pieces!</span>")
+	playsound(src, 'sound/items/smolebuildingdestoryedshort.ogg', 50, 1, -1, volume_channel = VOLUME_CHANNEL_MASTER)
+	new /obj/item/stack/material/smolebricks(loc)
+	new /obj/item/stack/material/smolebricks(loc)
+	qdel(src)
+	return
+
+//buildings
+/obj/structure/smoleruins
+	icon = 'icons/vore/smoleworld_vr.dmi'
+	icon_state = "ruins"
+	density = 0
+
+/obj/structure/smolebuilding/houses
+	name = "smole houses"
+	icon = 'icons/vore/smoleworld_vr.dmi'
+	icon_state = "smolehouses"
+
+/obj/structure/smolebuilding/business
+	name = "smole business"
+	icon = 'icons/vore/smoleworld_vr.dmi'
+	icon_state = "smolebusiness"
+
+/obj/structure/smolebuilding/warehouses
+	name = "smole warehouses"
+	icon = 'icons/vore/smoleworld_vr.dmi'
+	icon_state = "smolewarehouses"
+
+/obj/structure/smolebuilding/musem
+	name = "smole musem"
+	icon = 'icons/vore/smoleworld_vr.dmi'
+	icon_state = "smolemusem"
+//
+//CAR STUFF < WILL BE MESSED WITH IN A LATER UPDATE COMMENTED OUT FOR NOW
+///obj/item/smolecar
+//	name = "toy car"
+//	desc = "A toy car ready to be snapped together and burn rubber!"
+//	icon = 'icons/vore/smoleworld_vr.dmi'
+//	icon_state = "smolecarA_folded"
+//	item_state = "smolecarA"
+//	unfolded_type = /obj/structure/bed/chair/wheelchair/smolecar
+//
+///obj/structure/smolecar
+//	name = "Toy Car"
+//	desc = "Lets burn rubber!"
+//	icon = 'icons/vore/smoleworld_vr.dmi'
+//	icon_state = "smolecarA"
+//	anchored = 0
+//	buckle_movable = 1
+//	var/carry_type = /obj/item/smolecar
+//
+///obj/structure/bed/chair/wheelchair/smolecar/can_buckle_check(mob/living/M, forced = FALSE)
+//	. = ..()
+//	if(.)
+//
+//		if(M.get_effective_size() > RESIZE_TINY)
+//			to_chat(M, SPAN_WARNING("You are to big to fit in \the [src]."))
+	//		. = FALSE
+//
+//
+//snack planets, currently just plopped out will be organized later.
+/obj/item/trash/Asteroidsmall
+	icon = 'icons/vore/smoleworld_vr.dmi'
+	icon_state = "sp_asteriodB"
+	name = "asteriod"
+	desc = "A solid chunk of candy crumb that looks like a asteriod."
+
+/obj/item/trash/Asteroidlarge
+	icon = 'icons/vore/smoleworld_vr.dmi'
+	icon_state = "sp_asteriodA"
+	name = "asteriod"
+	desc = "A solid chunk of candy crumb that looks like a asteriod."
+
+/obj/item/trash/Asteroidmulti
+	icon = 'icons/vore/smoleworld_vr.dmi'
+	icon_state = "sp_asteriodC"
+	name = "asteriod"
+	desc = "Several chunks of sugar crumbs that looks like asteriods."
+
+/obj/item/weapon/bikehorn/tinytether
+	icon = 'icons/vore/smoleworld_vr.dmi'
+	icon_state = "tether_trash"
+	name = "tether"
+	desc = "Its a tiny bit of plastic in the shape of the tether. There seems to be a small button on top."
+
+/obj/item/weapon/bikehorn/tinytether/attack_self(mob/user as mob)
+	if(spam_flag == 0)
+		spam_flag = 1
+		playsound(src, 'sound/items/tinytether.ogg', 30, 1, volume_channel = VOLUME_CHANNEL_MASTER)
+		src.add_fingerprint(user)
+		spawn(20)
+			spam_flag = 0
+	return

```

Comment:
```suggestion
	honk_sound = 'sound/items/tinytether.ogg'
```
Seems odd to resort to copy-paste when there's a var for this.

Reply:
I have the code open like that due to not wanting it to be as loud as the clown horn. Also possibly leaving it open to have spaming lowered due to just what sound it makes. It makes the announcement sound but pitch shifted.